grakn-test/test-integration/src/test/java/ai/grakn/test/graql/shell/GraqlShellIT.java

```diff
@@ -71,8 +71,7 @@
 public class GraqlShellIT {
 
     @ClassRule
-    public static final DistributionContext dist = DistributionContext.startInMemoryEngineProcess().inheritIO(true);
-
+    public static final DistributionContext dist = DistributionContext.startInMemoryEngineProcess().inheritIO(false);
```

Comment:
I don't think we need this change anymore, since we suppressed the logs above.

Reply:
yes you are correct